multiversx_sdk_cli/delegation/staking_provider.py

```diff
@@ -10,440 +9,847 @@
 )
 from multiversx_sdk.abi import BigUIntValue, Serializer
 
+from multiversx_sdk_cli.base_transactions_controller import BaseTransactionsController
 from multiversx_sdk_cli.config import get_address_hrp
 from multiversx_sdk_cli.errors import BadUsage
 from multiversx_sdk_cli.interfaces import IAccount
-from multiversx_sdk_cli.validators.validators_file import ValidatorsFile
 
+DELEGATION_MANAGER_SC_ADDRESS_HEX = "000000000000000000010000000000000000000000000000000000000004ffff"
```

Comment:
I think the constant is defined somewhere else, as well (when used for creating staking providers).

Reply:
it is not defined anywhere else.